packages/waku/src/router/client.ts

```diff
@@ -371,10 +371,10 @@ const InnerRouter = ({ routerData }: { routerData: RouterData }) => {
   }, [changeRoute]);
 
   useEffect(() => {
-    const callback = (pathname: string, searchParamsString: string) => {
+    const callback = (path: string, query: string) => {
       const url = new URL(window.location.href);
-      url.pathname = pathname;
-      url.search = searchParamsString;
+      url.pathname = path;
+      url.search = query;
```

Comment:
This is just a refactor unrelated to this PR. (But found it by searching `searchParamsString`.)